config/sequelize.js

```diff
@@ -1,43 +1,47 @@
-exports.default = { 
+var sequelizeConfig = require('./sequelize.json');
```

Comment:
No, I don't want to change the config for this plugin to `.json`.  The point of having all config as `.js` files is that it allows more logic (reading from ENV, etc).